### ISSUE TITLE
[mesheryctl] Improve UX in `system config`

### ISSUE THREAD

**Issue by piyushsingariya** [meshery/meshery#4810]
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

#### Current Behavior

Current behavior is this
```
$ mesheryctl system config aks
Missing Meshery config file.
Create default config now [y/n]? y
Default config file created at /Users/me/.meshery/config.yaml
{
  "azure-cli": "2.24.1",
  "azure-cli-core": "2.24.1",
  "azure-cli-telemetry": "1.0.6",
  "extensions": {
    "azure-batch-cli-extensions": "6.0.0",
    "storage-preview": "0.7.2"
  }
}
Configuring Meshery to access AKS...
Please enter the Azure resource group name:
me-rg
Please enter the AKS cluster name:
me-test-001-aks
Merged “me-test-001-aks" as current context in /Users/me/.meshery/kubeconfig.yaml
Error getting context: could not read token:: open /Users/me/.meshery/auth.json: no such file or directory
```

#### Desired Behavior

As it is pretty visible that the error isn't helping the user here, the error should be brief enough to explain how to authenticate the mesheryctl i.e. through `system login`

Also a few steps were executed even though mesheryctl wasn't authenticated. The command shouldn't execute at all if not authenticated.

#### Mockups

---

#### Contributor Resources

- ⌨️ [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

**Comment by hexxdump** [comment]
I will add the checks to see whether the authentication succeeded or not(token present or not) and throw proper informational message to the user.

**Comment by leecalcote** [comment]
Wonderful, @hexxdump 🙂

**Comment by hexxdump** [comment]
Fixed via #4887
